docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -108,10 +108,11 @@ It provides a simple migration by generating Mimir configuration from Cortex con
 
 [`mimirtool`]({{< relref "../operators-guide/tools/mimirtool.md" >}}) provides a command for converting Cortex configuration to Mimir configuration that you can use to update both flags and configuration files.
 
-### Download mimirtool
+### Install mimirtool
```

Comment:
Can you check to see if any links are broken as a result of this change?

Reply:
The doc-validator tool also check anchors. Since CI has passed, no link should be broken.